pom.xml

```diff
@@ -84,6 +84,7 @@ under the License.
     <dependency>
       <groupId>org.apache.maven.plugin-tools</groupId>
       <artifactId>maven-plugin-annotations</artifactId>
+      <version>3.15.1</version>
```

Comment:
Should be the same version as `maven-plugin-plugin`, so we can use property `version.maven-plugin-tools`